ui/src/main/resources/Meeting/Code/MeetingSheet.xml

```diff
@@ -112,7 +112,9 @@
         #set ($title = $doc.getValue('meetingTitle'))
       #end
       #set ($meetingTitleId = 'Meeting.Code.MeetingClass_0_meetingTitle')
-      ; &lt;label for=""&gt;$!services.icon.render('font') $services.localization.render('contrib.meeting.field.meetingTitle')&lt;/label&gt;
+      ; &lt;label for=""&gt;
```

Comment:
the value in the `for` attribute is empty and the label won't work properly.

Reply:
The code from https://github.com/xwikisas/application-meeting/commit/1a3b93ee04e85af44ef50967d1e9c68cbf8dc8ea wasn't fetched. When you start working on something, make sure you got the latest changes from github.